### DISCUSSION TITLE
Mobile headings

### DISCUSSION THREAD

**Discussion by wa-rren-dev** [nhsevidence/NICE-Experience#142] [Ideas]
Reduce font size and possibly line-height for H1, investigate other headings  - guidance titles are long, take up a lot of space on Mobile screens. 

![image](https://user-images.githubusercontent.com/6341474/81273488-da221a80-9046-11ea-95fe-24386740edf7.png)

**Comment by rnickr** [comment]
This has been addressed with the updated typography settings that were done as part of the rebrand work
